CONTRIBUTING.md

```diff
@@ -48,14 +56,20 @@ cd aider
 
 ### Create a Virtual Environment
 
-It is recommended to create a virtual environment outside of the repository to keep your development environment isolated.
+It is recommended to create a virtual environment outside of the repository to keep your development environment isolated. To create one, use `venv` or `uv`.
 
 #### Using `venv` (Python 3.9 and later)
 
 ```
 python -m venv /path/to/venv
 ```
 
+### Using `uv`
+
+```
+uv venv /path/to/venv
```

Comment:
One of the biggest benefits of `uv` is that you don't have to think about virtual environments at all, since `uv` manages it automatically. `uv venv` is largely intended for legacy workflows. There is no need to scaffold, activate, or deactivate virtual environments anymore.

I suggest avoiding the `uv venv` and `uv pip` commands for this reason.